buildkite/bazelci.py

```diff
@@ -1568,6 +1572,37 @@ def SetEnvVar(config_key, env_var_name):
             )
         )
 
+    if VALIDATE_CONFIG_OPTION in configs:
+        output = execute_command_and_get_output(
+            [
+                "git",
+                "diff-tree",
+                "--no-commit-id",
+                "--name-only",
+                "-r",
+                os.getenv("BUILDKITE_COMMIT"),
+            ]
+        )
+        config_files = [
+            l
+            for l in output.split("\n")
+            if l.startswith(".bazelci/") and os.path.splitext(l)[1] in CONFIG_FILE_EXTENSIONS
```

Comment:
os.path.splitext(l)[1] will throw an exception if someone adds a file without an extension to the directory, right?

Reply:
Nope, the second element is an empty string in that case.